Classes/Wwwision/Neos/MailChimp/Domain/Service/MailChimpService.php

```diff
@@ -24,17 +25,23 @@ class MailChimpService
     /**
      * @var RequestEngineInterface
      */
-    private $requestEngine;
+    protected $requestEngine;
 
     /**
      * @var string
      */
-    private $apiKey;
+    protected $apiKey;
 
     /**
      * @var string
      */
-    private $apiEndpoint;
+    protected $apiEndpoint;
+
+    /**
+     * @Flow\Inject(lazy=false)
+     * @var VariableFrontend
+     */
+    protected $cache;
```

Comment:
I tried to avoid the "magic property injection" in this class.
If we use a cache we should constructor-inject it like the other dependencies